README.md

```diff
@@ -9,10 +9,17 @@ allow for customization for new fields and extensibility to new services. Using
 this library will allow you to easily configure OTA updates from a variety of
 sources and keep OTA functionality separated from any on-device application.
 
-This repository contains an example of an OTA orchestrator using FreeRTOS and
-coreMQTT. The example will check IoT Core for an existing OTA Job, download its
-associated file, output the file over the command line, and report success back
-to IoT Core.
+This repository contains the following two example OTA orchestrators. Both
+orchestrators use FreeRTOS, coreMQTT and IoT Jobs library.
+
+1. **Simple OTA Orchestrator**: It is a simple orchestrator which checks IoT Core
+for an existing OTA Job, download its associated file, output the file over the
+command line, and report success back to IoT Core.
+2. **OTA Agent Orchestrator**: The OTA agent orchestrator operates by managing
```

Comment:
I'd mention that this example is designed to mimic the OTA agent found in the old OTA repository

Reply:
Added this in the README.